style.css

```diff
@@ -635,3 +678,106 @@ main{
     border-radius: 50%;
 }
 
+/* Rating */
+
+.rating-container{
+    /* border: 1px solid var(--secondary-grey); */
+    padding: 1rem;
+    background-color: black;
+    border-radius: 10px;
+}
+
+.rating-star{
+    padding: 0.5rem;
+    cursor: pointer;
+    color: white;
+}
+
+.rating-star:hover{
+    color: var(--light-yellow-outlined);
+}
+
+.rating-active{
+    color: var(--light-yellow-outlined);
+}
+
+.rating-inactive{
+    color: white;
+}
+
+#rating-text{
+    width: 30%;
+    height: 2rem;
+    font-weight: bold;
+    color: var(--primary-maroon);
+}
+/* Grid */
+
+.grid{
+    width: 70%;
+    display: grid;
+    gap: 0.5rem;
+    border: 1px solid var(--secondary-grey);
+    padding: 0.5rem;
+    font-size: 1.5rem;
+}
+
+.grid-container-2-cols{
+    grid-template-columns: 1fr 1fr;
+}
+
+.grid-container-3-cols{
+    grid-template-columns: 1fr 1fr 1fr;
+}
+
+.grid1{
+    background-color: var(--primary-maroon);
+    color: var(--secondary-grey);
+    height: 15rem;
+    width: 100%;
+    display: grid;
+    place-content: center;
+}
+
+.grid2{
+    background-color: var(--secondary-grey);
+    color: var(--dark-maroon);
+    height: 15rem;
+    width: 100%;
+    display: grid;
+    place-content: center;
+}
+
+.grid3{
+    border: 1px solid var(--primary-maroon);
+    color: var(--primary-maroon);
+    box-sizing: border-box;
+    height: 15rem;
+    width: 100%;
+    display: grid;
+    place-content: center;
+}
+/* Toast */
+
+.toast-container{
+    width: 60%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0 1rem;
+    margin: 0.5rem 0;
+    border-radius: 5px;
+}
+
+/* List */
+
+.list-stacked{
+    background-color: var(--secondary-grey);
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    border: 1px solid var(--light-grey);
```

Comment:
Variables `secondary-grey` and  `light-grey` might confuse people due to similar names. You can use names like `grey-100`, `grey-400` to denote darkness.

Reply:
Okay, good point, will keep that in mind and try to change accordingly later